### ISSUE TITLE
Misautocorrection by Style/Block

### ISSUE THREAD

**Issue by mvz** [rubocop/rubocop#1441]
In autocorrect mode, the Style/Block cop corrects the following code:

``` ruby
foo :bar, :baz, qux: lambda { |a|
  bar a
}
```

The result is:

``` ruby
foo :bar, :baz, qux: lambda do |a|
  bar a
end
```

This gives an error in Ruby:

``` shell
[matijs@walnut]  ruby test_rubocop.rb
test_rubocop.rb:1:in `lambda': tried to create Proc object without a block (ArgumentError)
    from test_rubocop.rb:1:in `<main>'
```

The Rubocop version output is:

``` shell
[matijs@walnut] rubocop -V
warning: parser/current is loading parser/ruby21, which recognizes
warning: 2.1.4-compliant syntax, but you are running 2.1.3.
0.27.1 (using Parser 2.2.0.pre.7, running on ruby 2.1.3 x86_64-linux)
```
